build.xml

```diff
@@ -791,6 +813,7 @@ ${git_status}
         <include name="**/*" />
       </fileset>
     </delete>
+    <delete dir="${srcdir}/themes/minktest" />
```

Comment:
Any reason not to use ${themedir} here for consistency?

Reply:
And a related note: maybe it's better to name this something like `testthemedir` or `dynamicThemeFixtureDir` to remove any ambiguity about its purpose or the reason for its deletion.

Reply:
`themedir` is just a local property in configure_install target, and I don't think it's that useful to define globally just for this.

Reply:
Ahh, sorry for misunderstanding the scope. In that case, all is well. :-)